src/client/components/app-header/app-header.vue

```diff
@@ -1,23 +1,26 @@
 <template>
-  <header role="banner" class="app-header">
+  <header role="banner" class="app-header" :class="{ 'app-header--sticky' : menu.isSticky }">
     <nuxt-link :to="localePath('index')" class="app-header__identity">
       <img class="app-header__logo" src="/images/logo.svg" alt="" />
-      <h1 class="app-header__title">Lean Web Kit</h1>
+      <h1 class="app-header__title">Lean Web Kit<br><small>a Voorhoede product</small></h1>
```

Comment:
Can you spread this over multiple lines for legibility? 
Also, consider adding a class to the `small`. I feel that the the text should not wrap, so there are some tweaks needed for responsive behaviour